yarn-error.log

```diff
@@ -0,0 +1,4980 @@
+Arguments: 
```

Comment:
please add this to the `.gitignore`:
1. delete the file
2. commit the change
3. add to `.gitignore`
4. commit the changes